app/Http/Controllers/APiSyncController.php

```
@@ -2,24 +2,119 @@
+use Illuminate\Support\Facades\Log;
 
 class APiSyncController extends Controller
 {
+    public function syncAll(Request $request)
+    {
+        try {
+            ApiHrmsSyncJob::dispatch('syncAll');
+            ApiInventorySyncJob::dispatch('syncAll');
+        } catch (\Exception $e) {
+            Log::error('Failed to dispatch HRMS and IMS sync job', ['error' => $e->getMessage()]);
+            throw new \Exception("HRMSand IMS sync failed: " . $e->getMessage());
```

PHPMD (2.15.0)

Missing class import via use statement (line '19', column '23'). (Clean Code Rules)

(MissingImport)